### ISSUE TITLE
Add Custom UI to new module in Nebular

### ISSUE THREAD

**Issue by AhsanNissar** [akveo/nebular#1991]
### Issue type

**I'm submitting a ...**  (check one with "x")

* [x] bug report
* [ ] feature request

### Issue description
I am trying to create a new module named student alongside pages module. I have created the new module in the same way as the pages module and generated a new component inside the student module. Now that i am trying to make use of url to redirect me directly on to the new component page but instead it shows me infinite spinner. I did go through some of the same issues and saw that i need to make use of nb-layout in html pages but i do not know in which html pages specifically. 
**Current behavior:**
Seeing infinite spinner. 
Normally the url is localhost:4200/pages/dashboard but mine is different since i have created a new module

**Expected behavior:**
Basically I want to create a custom UI for my components in student module so i don't want anything related to Nebular in that module. How can i do that?

**Steps to reproduce:**
Create a new module alongside pages module and generate a component in that module and redirect to it using its url

![Capture](https://user-images.githubusercontent.com/14846851/65962991-41556c00-e473-11e9-9096-b79214ffa875.JPG)
